shared/fs/header/breadcrumb-popup.js

```diff
@@ -33,14 +34,13 @@ const BreadcrumbPopupMenu = ({routeProps}: PopupMenuProps) => {
     title: i.name,
     view: itemView(i.name, isTeamPath && idx === items.length - 3),
   }))
-  return <PopupMenu items={popupItems} style={stylePopup} onHidden={onHidden} />
+  return <ModalLessPopupMenu items={popupItems} style={stylePopup} onHidden={onHidden} />
 }
 
 const stylePopup = {
-  position: 'absolute',
-  left: -28,
-  top: 24,
-  width: 196,
+  width: isMobile ? '100%' : 196,
+  marginTop: isMobile ? undefined : 12,
+  marginLeft: isMobile ? undefined : -12,
```

Comment:
Could use `platformStyles` here? Not critical though so feel free to delay it until your next PR.

Reply:
Platform styles isn't really required if your styles are compatible with being cross platform. Mainly used when that's not true

Reply:
Oh I thought that was just a cleaner way to do things. Thanks for the clarification.

Reply:
I think it is kinda cleaner and it'll match more code but it's not strictly required. In other places it is